src/Service/PdfCreator.php

```diff
@@ -4,38 +4,39 @@
 
 namespace App\Service;
 
-use Spipu\Html2Pdf\Html2Pdf;
+use Dompdf\Dompdf;
+use Dompdf\Options;
 
 final class PdfCreator implements PdfCreatorInterface
 {
     private $pdf;
     private $orientation;
     private $format;
-    private $lang;
-    private $unicode;
-    private $encoding;
-    private $margin;
 
-    public function __construct(string $orientation, string $format, string $lang, bool $unicode, string $encoding, array $margin)
+    public function __construct(string $orientation, string $format)
```

Comment:
You should create value objects for `$orientation` and `$format`, allowing only valid values.